Code/Framework/AzFramework/AzFramework/Entity/GameEntityContextComponent.cpp

```diff
@@ -319,6 +285,17 @@ namespace AzFramework
             EBUS_EVENT_RESULT(currentEntity, AZ::ComponentApplicationBus, FindEntity, *entityIdIter);
             if (currentEntity)
             {
+                bool isPrefabSystemEnabled = false;
+                AzFramework::ApplicationRequests::Bus::BroadcastResult(
+                    isPrefabSystemEnabled, &AzFramework::ApplicationRequests::IsPrefabSystemEnabled);
+                if (isPrefabSystemEnabled)
+                {
+                    SpawnedEntityTicketMapperInterface* spawnedEntityTicketMapperInterface =
+                        AZ::Interface<SpawnedEntityTicketMapperInterface>::Get();
```

Comment:
Sorry, I thought this was also used by the editor in places. There's still an issue though as there's still the plan to support multiple viewports with different game previews running, so long term this is more work that would need to be moved to AzFramework::Scene so better to do it right now instead.

Reply:
This interface is not used anymore since I changed it to store the ticket inside the entity now. If anything still can be moved to AzFramework::Scene, let me know. Although, I'll probably do it as a follow up PR to keep the scope of this PR from growing